plugins/blockvolume/blockprovider/gluster-virtblk/glustervirtblk.go

```diff
@@ -17,28 +17,28 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
-const providerName = "gluster-loopback"
+const providerName = "gluster-virtblk"
```

Comment:
Remove `gluster-` prefix and expand blk to block, so that in URL we can use `/v1/blockvolumes/virtblock`

Reply:
done